docs/developer-docs/latest/setup-deployment-guides/installation/docker.md

```diff
@@ -6,6 +6,14 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guid
 
 # Installing using Docker
 
+:::caution
+This Docker image is only for Strapi v3. For now, Strapi will not update the image for v4.
+However, to build an image compatible with Strapi v4, we recommend following this guide by Simen Daehlin, Community Star at Strapi, <https://blog.dehlin.dev/docker-with-strapi-v4>.
```

Comment:
Just wondering if we should keep the link as-is, or if we should write something like "[the guide written by …](link to guide)" 🤔

Reply:
I agree @pwizla I think it would be better to follow the usual formatting for links, instead of directly pasting the URL.

Reply:
The link style is updated.